src/viam/ur/module/ur_arm.cpp

```diff
@@ -562,10 +588,18 @@ void URArm::reconfigure(const Dependencies& deps, const ResourceConfig& cfg) {
 std::vector<double> URArm::get_joint_positions(const ProtoStruct&) {
     const std::shared_lock rlock{config_mutex_};
     check_configured_(rlock);
-    return get_joint_positions_(rlock);
+    auto curr_joint_position_rad = get_joint_positions_rad(rlock);
+    std::vector<double> joint_position_degree;
+    joint_position_degree.reserve(curr_joint_position_rad.size());
+    std::transform(std::make_move_iterator(curr_joint_position_rad.begin()),
+                   std::make_move_iterator(curr_joint_position_rad.end()),
+                   std::back_inserter(joint_position_degree),
+                   radians_to_degrees<double>);
+
+    return joint_position_degree;
```

Comment:
If you want to be "range-y":
```
    const auto degrees = radians | boost::adaptors::transformed([](const auto& r) { return radians_to_degrees(r); });
    return {begin(degrees), end(degrees)};
```

I think, since `vector6d` has random access iterators, the range does too, so this should also be O(1) distance calculation and I expect the vector constructor to leverage that to call `reserve` automatically before pushing the items.